smrf/distribute/wind/wind.py

```diff
@@ -111,10 +120,14 @@ def initialize(self, topo, data):
         """
 
         self._logger.debug('Initializing distribute.wind')
-
+        self.date_time = date_time
         self.wind_model._initialize(topo, data.metadata)
 
-        if self.config['wind_model'] != 'interp':
+        if self.check_wind_model_type('winstral'):
+            self.add_thread_variables(
+                list(self.wind_model.BASE_THREAD_VARIABLES))
```

Comment:
The first access  of `self.wind_model.thread_variables` should do this for you without the need to set this beforehand.

Reply:
Moved this to a property of the wind model